kubernetes/crd/domain-crd.yaml

```diff
@@ -3585,6 +3585,43 @@ spec:
                       enum:
                       - RUNNING
                       - ADMIN
+                    replicas:
+                      description: The number of cluster member Managed Server instances
+                        to start for this WebLogic cluster. The operator will sort
+                        cluster member Managed Server names from the WebLogic domain
+                        configuration by normalizing any numbers in the Managed Server
+                        name and then sorting alphabetically. This is done so that
+                        server names such as "managed-server10" come after "managed-server9".
+                        The operator will then start Managed Server instances from
+                        the sorted list, up to the `replicas` count, unless specific
+                        Managed Servers are specified as starting in their entry under
+                        the `managedServers` field. In that case, the specified Managed
+                        Server instances will be started and then additional cluster
+                        members will be started, up to the `replicas` count, by finding
+                        further cluster members in the sorted list that are not already
+                        started. If cluster members are started because of their related
+                        entries under `managedServers`, then this cluster may have
+                        more cluster members running than its `replicas` count. Defaults
+                        to `spec.replicas`, which defaults 0.
+                      type: number
+                      minimum: 0.0
+                    domainUID:
```

Comment:
Is this temporary? (I understand that the Clusters schema needs to have a reference to the domain -- I'm referring to that content being in the domains.spec.clusters schema)

Reply:
Yes, this is temporary.  I added domainUid to the Cluster spec included in the Cluster resource.  The Cluster 'spec' is still in the Domain CRD and will remove it in the next set of changes as we transition the configuration from Domain resource to the Cluster resource.